public/electron.ts

```diff
@@ -66,12 +73,26 @@ function createMenu() {
                 { role: 'minimize' },
                 { role: 'close' }
             ]
+        },
+        {
+            label: 'Help',
+            submenu: [
+                {
+                    accelerator: 'CommandOrControl+Shift+I',
```

Comment:
is it the default shortcut that people follow for electron apps?

Reply:
It's the shortcut that VSCode uses.